doc/guide.en.rst

```diff
@@ -1,286 +0,0 @@
-.. encoding: utf-8
-
-Developer's guide
```

Comment:
Will this help to keep the history of changes?
```
$ git mv doc/guide.en.rts doc/dev-guide.rst
```

Reply:
rts -> rst